app/controllers/state_file/archived_intakes/archived_intake_controller.rb

```diff
@@ -6,18 +6,32 @@ def current_request
         StateFileArchivedIntakeRequest.find_by(ip_address: ip_for_irs, email_address: session[:email_address])
       end
 
+      def current_archived_intake
+        current_request.state_file_archived_intake
+      end
+
       def create_state_file_access_log(event_type)
         StateFileArchivedIntakeAccessLog.create!(
           event_type: event_type,
           state_file_archived_intake_request: current_request
         )
       end
 
+      def address_challenge_set
```

Comment:
why is the address_challenge_set defined here? are we trying to hide the method? 

🤔 if that's the case, since it's not shared across controllers, how do you feel about putting it at the model level?